cypress/integration/capture.spec.js

```diff
@@ -194,6 +196,45 @@ describe('Event capture', () => {
         })
     })
 
+    describe('capturing performance', () => {
+        it('sends performance timing with the pageview when enabled', () => {
+            given('options', () => ({ capture_pageview: true, capture_performance: true }))
+            start()
+
+            // Pageview will be sent immediately
+            cy.wait('@capture').its('request.headers').should('deep.equal', {
+                'Content-Type': 'application/x-www-form-urlencoded',
+            })
+            cy.get('@capture').should(({ request }) => {
+                const captures = getBase64EncodedPayload(request)
+
+                expect(captures['event']).to.equal('$pageview')
+
+                expect(captures.properties).to.have.property('performance')
+                expect(captures.properties.performance).to.have.property('navigation')
```

Comment:
Cypress tests run in chrome and we can make use of that fact here given that the contents of the performance keys are really opaque (when do we know it breaks?)

I think we can make this test more useful by asserting that each of the properties contains at least some keys that will be later used in analysis.

Reply:
have added some more detail to the tests and a comment on why I'm being slightly cautious